.github/integrationBenchScripts/cloud-tests-local-trigger.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -eo pipefail
+
+cd "$(dirname "$0")" || exit 1
+
+GITHUB_RUN_ID=$1
+REPOSITORY_URL=$2
+BRANCH_NAME=$3
+PROVER=$4
+DEGREE=19
+
+PROVER_INSTANCE=$(cat "$HOME/CI_Github_Trigger/$GITHUB_RUN_ID/prover_instance")
+echo "Prover instance at trigger: $PROVER_INSTANCE"
+
+export PROVER_IP=$(tccli cvm DescribeInstances --InstanceIds "[\"$PROVER_INSTANCE\"]" | grep -A 1 PublicIpAddress | egrep -o '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}')
+echo "Prover IP: $PROVER_IP"
+
+rm ~/.ssh/known_hosts*
+
+prepare_env() {
+  ssh -i ~/.ssh/bench.pem -o StrictHostKeyChecking=no ubuntu@"$PROVER_IP" "bash -s" -- <../weeklyBenchScripts/00_installGo.sh
+  ssh -i ~/.ssh/bench.pem -o StrictHostKeyChecking=no ubuntu@"$PROVER_IP" "bash -s" -- <../weeklyBenchScripts/00_installRust.sh
+  ssh -i ~/.ssh/bench.pem -o StrictHostKeyChecking=no ubuntu@"$PROVER_IP" "bash -s" -- <../weeklyBenchScripts/01_installDeps.sh
+  ssh -i ~/.ssh/bench.pem -o StrictHostKeyChecking=no ubuntu@"$PROVER_IP" "bash -s" -- <../weeklyBenchScripts/02_setup.sh
+}
+
+prepare_repo() {
+  ssh -i ~/.ssh/bench.pem -o StrictHostKeyChecking=no ubuntu@"$PROVER_IP" "bash -s" -- "$GITHUB_RUN_ID" <../weeklyBenchScripts/03_prepareProver.sh
+  ssh -i ~/.ssh/bench.pem -o StrictHostKeyChecking=no ubuntu@"$PROVER_IP" "bash -s" -- "$GITHUB_RUN_ID" "$REPOSITORY_URL" "$BRANCH_NAME" <../integrationBenchScripts/04_clone.sh
+#  ssh -i ~/.ssh/bench.pem -o StrictHostKeyChecking=no ubuntu@"$PROVER_IP" "bash -s" -- "$GITHUB_RUN_ID" <../weeklyBenchScripts/05_build.sh
+  ssh -i ~/.ssh/bench.pem -o StrictHostKeyChecking=no ubuntu@"$PROVER_IP" "bash -s" -- <../weeklyBenchScripts/06_rsSysstat.sh &
+  sleep 5
+
+  ssh -i ~/.ssh/bench.pem -o StrictHostKeyChecking=no ubuntu@"$PROVER_IP" "bash -s" -- "$DEGREE" "$GITHUB_RUN_ID" "$PROVER" <../integrationBenchScripts/07_execBench.sh
+  declare -g RESULT=$?
+  chmod u+x ../integrationBenchScripts/08_processResults.sh
+  ../integrationBenchScripts/08_processResults.sh "$PROVER" "$DEGREE"
+}
+
+prepare_env
+prepare_repo
+
+kill_ssh() {
```

Comment:
This probably needs an explanation to why kill_ssh is needed and if it could affect a maintenance/debugging session opened that can be killed by inadvertance.

Reply:
I agree.